src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -151,9 +147,15 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, bool display
 
             if (mirrorData.IsCollection)
             {
+                int count = 0;
                 foreach (var el in mirrorData.GetElements())
                 {
-                    GetRenderPackagesFromMirrorData(el, displayLabels, isNodeSelected, ref labelMap, ref count);
+                    if (el.IsCollection || el.Data is IGraphicItem)
```

Comment:
Will there be a case when el.Data is IEnumerable, probably not, because IEnumerable will always be marshaled as a collection??